week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(productPrice) {
+  let tax = productPrice * 0.2 + productPrice;
```

Comment:
Good job! How could you make this simpler by using the variable `productPrice` only once in this statement?